rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -304,6 +290,119 @@ static bool map_ext_to_ds3_input(const u32 port_no, CellGemExtPortData& ext)
 	return true;
 }
 
+/**
+ * \brief Maps Move controller data (digital buttons, and analog Trigger data) to mouse input.
+ *        Move Button: Mouse1
+ *        Trigger:     Mouse2
+ * \param mouse_no Mouse index number to use
+ * \param digital_buttons Bitmask filled with CELL_GEM_CTRL_* values
+ * \param analog_t Analog value of Move's Trigger.
+ * \return true on success, false if mouse mouse_no is invalid
+ */
+static bool mouse_input_to_pad(const u32 mouse_no, be_t<u16>& digital_buttons, be_t<u16>& analog_t)
+{
+	const auto handler = g_fxo->get<MouseHandlerBase>();
+
+	std::scoped_lock lock(handler->mutex);
+
+	if (!handler || mouse_no >= handler->GetMice().size())
+	{
+		return false;
+	}
+
+	memset(&digital_buttons, 0, sizeof(digital_buttons));
+
+	const auto& mouse_data = handler->GetMice().at(0);
+
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_1 && mouse_data.buttons & CELL_MOUSE_BUTTON_2)
+		digital_buttons |= CELL_GEM_CTRL_CIRCLE;
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_3)
+		digital_buttons |= CELL_GEM_CTRL_CROSS;
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_2)
+		digital_buttons |= CELL_GEM_CTRL_MOVE;
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_1 && mouse_data.buttons & CELL_MOUSE_BUTTON_3)
+		digital_buttons |= CELL_GEM_CTRL_START;
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_1)
+		digital_buttons |= CELL_GEM_CTRL_T;
+	if (mouse_data.buttons & CELL_MOUSE_BUTTON_2 && mouse_data.buttons & CELL_MOUSE_BUTTON_3)
+		digital_buttons |= CELL_GEM_CTRL_TRIANGLE;
+
+	analog_t = mouse_data.buttons & CELL_MOUSE_BUTTON_1 ? 0xFFFF : 0;
```

Comment:
Use () around the button check

Reply:
ah, i have just set same master, is velo code, not touched this